### PR TITLE
Ramazan Ince Database HW1

### PR DESCRIPTION
**Pull request by RamazanInce**

### CODE REVIEW

File: Week1/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'userdb',
+});
+
+connection.connect();
+const allDatabase = 'CREATE DATABASE IF NOT EXISTS world';
+const useDatabase = 'USE world';
```
Comment (harditsingh) [inline Week1/index.js]: If you look at the world.sql file, the data is being loaded in a database called 'new_world', so in order to get your queries to work, you will have to use database 'new_world' instead in both of the above lines. For more information on how to load the data into the tables, [check here](https://github.com/HackYourFuture/databases/tree/master/Week2).

Comment (RamazanInce) [reply Week1/index.js]: Hi @harditsingh thanks for your comments. I would like to do what you request but you closed it then I can not update. Can you open it for me to update, if it is requested.

Thanks for your effort...

Comment (harditsingh) [reply Week1/index.js]: I opened it again so you can work on it, best of luck!

**Comment by unmeshvrije** [comment]
Already reviewed.

**Comment by unmeshvrije** [comment]
reopening for work

**Comment by wouterkleijn** [comment]
Thanks @harditsingh @RamazanInce Looking forward to the changes

**Pull request closed without merging**